pkg/tests/mappings/manifest_test_lib.py

```diff
@@ -32,10 +32,10 @@ def assertManifestsMatch(self, expected, got):
     """
```

Comment:
While you're in the neighborhood, consider setting `self.maxDiff` to something large (or `None`) in `setUp` to make any large differences more apparent.

Reply:
I want to change this to a more useful semantic diff. So I'll defer on that.